pptx_blueprint/__init__.py

```diff
@@ -54,7 +54,27 @@ def _find_shapes(self, label: str) -> Iterable[BaseShape]:
         Args:
             label (str): label of the placeholder (without curly braces)
         """
-        pass
+        slide_number, tag_name = label.split(":")
+        matched_shapes = []
+
+        def _find_shapes_in_slide(slide):
+            for shape in slide.shapes:
```

Comment:
This looks like a great change to use a more functional approach like 'filter' :)

Reply:
meant 'chance' instead of 'change', of course

Reply:
thanks